sql/000_base_structure.sql

```diff
@@ -142,7 +142,7 @@ CREATE TABLE IF NOT EXISTS `settings` (
   UNIQUE KEY `setting` (`name`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 
-INSERT INTO `settings` (`name`, `value`) VALUES ('DB_VERSION', '1.0.1');
+INSERT INTO `settings` (`name`, `value`) VALUES ('DB_VERSION', '1.0.2');
```

Comment:
You'll also have to define the expected version via: https://github.com/MPOS/php-mpos/blob/master/include/version.inc.php#L5

I did see you supplied the upgrade script, so that's great - just also add the new version to the include file so the admin dashboard properly displays the version change :-)